inc/functions/admin.php

```diff
@@ -526,3 +526,31 @@ function rocket_create_options_hash( $value ) {
 
 	return md5( wp_json_encode( $value_diff ) );
 }
+
+/**
+ * This function returns the license type for a customer.
+ *
+ * @param object $customer_data customer data as an object.
+ * @return string
+ */
+function rocket_get_license_type( $customer_data ) {
+	$type = '';
+
+	if ( false === $customer_data
+		|| ! isset( $customer_data->licence_account )
+	) {
```

Comment:
Let's make it more readable.

```suggestion
	if (
		false === $customer_data
		||
		! isset( $customer_data->licence_account )
	) {
```

Reply:
I've corrected it.